sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducer/EventHubProducerLiveTests.cs

```diff
@@ -145,7 +145,7 @@ await using (var producer = client.CreateProducer())
         /// </summary>
         ///
         [Test]
-        public async Task ProducerCanSendEventsUsingAPartitionHashHey()
+        public async Task ProducerCanSendEventsUsingAPartitionHashKey()
```

Comment:
I swear I can't see what's different.

Reply:
Spotted it! Hey => Key.

Reply:
Every once in a while, I find my own typos.